### DISCUSSION TITLE
有交流群吗？

### DISCUSSION THREAD

**Discussion by xmsz-stu** [web-infra-dev/modern.js#6081] [Q&A]
问issue好像回复的很少，有没有交流群可以互相回答一下

**Comment by xmsz-stu** [comment]
哭了

**Comment by xc2** [comment]
kfll invited you to a group on Feishu, click https://applink.feishu.cn/client/chat/chatter/add_by_link?link_token=261hc013-9645-4216-be4f-fe42753ab6a4 to join！

**Reply by xmsz-stu** [reply]
感恩 这个我之前进了 是rspack的 也可以在里面讨论modern.js吗
